RELEASE_NOTES.md

```diff
@@ -101,16 +103,16 @@ accompanying blog post.
   which is a standard JPEG.
 
 - The preference to auto-apply a sharpening has been removed. One can
-  add an auto-applied presets in the sharpen module and gain the same
+  add an auto-applied preset in the sharpen module and gain the same
   effect.
 
 - Add support for updating existing pictures in Piwigo export.
 
 - The White Balances presets have been migrated to an external JSON file.
 
 - Remove Color Balance layout and White Balance control background
-  color from the preference as this can be done directly on the module
-  itself.
+  color from preferences as this can be done directly on the modules
```

Comment:
**in** / **on** the modules?

Reply:
"within the modules", probably. Also, "White Balances presets" is wrong (a bit of Franglais I think), should be "White Balance presets"